### PR TITLE
Provide better labels for left/right arrows

### PR DESCRIPTION
**Pull request by mejackreed**
Fixes https://github.com/sul-dlss/exhibits/issues/1774

In conjunction with https://github.com/projectblacklight/blacklight/pull/2253

Screen Shot 2020-03-02 at 6 09 51 PM

### CODE REVIEW

File: config/locales/blacklight-gallery.en.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 en:
   blacklight:
+    icon:
+      chevron_left_previous_image: 'previous image' 
+      chevron_right_next_image: 'previous image'
```
Comment (cbeer) [inline config/locales/blacklight-gallery.en.yml]:
```suggestion
      chevron_right_next_image: 'next image' 
```

File: app/views/catalog/_slideshow.html.erb [hunk 2]
```diff
@@ -6,10 +6,10 @@
 
     <!-- Controls -->
     <a class="left carousel-control prev" href="#slideshow" data-slide="prev">
-      <%= blacklight_icon 'chevron_left' %>
+      <%= blacklight_icon 'chevron_left', { additional_options: { label_context: 'previous_image' }} %>
```
Comment (mejackreed) [inline app/views/catalog/_slideshow.html.erb]: @cbeer @jcoyne slight change here for backwards compatibility and hopefully future proofing

Comment (mejackreed) [reply app/views/catalog/_slideshow.html.erb]: 😢 I'm not sure how to make this go green without a Blacklight release. Any suggestions?

Comment (jkeck) [reply app/views/catalog/_slideshow.html.erb]: Not w/o adding some sort of wrapper that checked the `blacklight_icon` method's `parameters` and then adding the `additional_options` conditionally.  Not sure if that's ultimately worth it 🤷‍♂️

Comment (mejackreed) [reply app/views/catalog/_slideshow.html.erb]: Yeah.. doesn't seem like it.

**Pull request merged**